components/form/util.ts

```diff
@@ -59,3 +60,20 @@ export function getStatus<DefaultValue>(
   }
   return status;
 }
+
+/**
+ * Get element inner text
+ */
+export function getTextFromElement(element: React.ReactElement): string {
```

Comment:
This will not work as expect since React support HOC:
```tsx
const My = () => 'Hello World';
```

`getTextFromElement` will get nothing.

Reply:
And this will also break in some locale case:
```tsx
const Counter = ({ unit }) => {
  const counter = useContext(parentContext);
  return `${counter} ${unit}s missing`;
};
```

get `unit` instead of `3 units missing`

Reply:
Another case is customize component use another prop instead of `children`:

```tsx
<Badge status="success" text="Success" />
```